admin/view/template/payment/razorpay_subscription/razorpay_plan_form.twig

```diff
@@ -0,0 +1,191 @@
+								<input type="text" name="billing_cycle"  placeholder="{{ entry_billing_cycle }}" value="{{ plan_bill_cycle }}" id="input-billing-cycle" class="form-control" />
+								{% if error_billing_cycle %}
+									<div class="text-danger">{{ error_billing_cycle }}</div>
+								{% endif %}
+							</div>
+					</div>
+					</div>
+					<div class="form-group required">
+						<div class="row mb-3">
+							<label class="col-sm-2 col-form-label" for="input-billing-amount">{{ entry_billing_amount }}</label>
+							<div class="col-sm-10">
+								<input type="text" name="billing_amount" placeholder="{{ entry_billing_amount }}"  value="{{ plan_bill_amount }}" id="input-billing-amount" class="form-control" />
+								{% if error_billing_amount %}
+									<div class="text-danger">{{ error_billing_amount }}</div>
+								{% endif %}
+							</div>
+						</div>
+					</div>
+					<div class="form-group ">
+						<div class="row mb-3">
+							<label class="col-sm-2 col-form-label" for="input-trial">{{ entry_trial }}</label>
+							<div class="col-sm-10">
+							<input type="text" name="plan_trial" placeholder="{{ entry_trial }}"  value="{{ plan_trial }}" id="input-trial" class="form-control" />
+							</div>
+						</div>
+					</div>
+					<div class="form-group ">
+						<div class="row mb-3">
+							<label class="col-sm-2 col-form-label" for="input-plan-addons">{{ entry_plan_addons }}</label>
+							<div class="col-sm-10">
+							<input type="text" name="plan_addons" placeholder="{{ entry_plan_addons }}"  value="{{ plan_addons }}" id="input-plan-addons" class="form-control" />
+							</div>
+						</div>
+					</div>
+					<div class="form-group required">
+						<div class="row mb-3">
+							<label class="col-sm-2 col-form-label" for="input-billing-cycle">{{ entry_plan_status }}</label>
+							<div class="col-sm-10">
+							<select name="plan_status" id="input-status" class="form-control">
+								{% if status %}
+									<option value="1" selected="selected">{{ text_enabled }}</option>
+									<option value="0">{{ text_disabled }}</option>
+								{% else %}
+									<option value="1">{{ text_enabled }}</option>
+									<option value="0" selected="selected">{{ text_disabled }}</option>
+								{% endif %}
+							</select>
+							</div>
+						</div>
+					</div>
+					<div class="float-end">
+						<button form="form-razorpay-addplan" data-toggle="tooltip" title="{{ button_save }}" class="btn btn-primary"><i class="fa fa-save"></i> {{ button_save }}</button>
+						<a href="{{ cancel }}" class="btn btn-default"><i class="fa fa-reply"></i> {{ button_cancel }}</a>
+					</div>
+				</form>
+			</div>	
+		</div>
+	</div>
+</div>
+<script type="text/javascript"><!--
+//auto load product id
+$('#input-product').autocomplete({
+	'source': function(request, response) {
+		$.ajax({
+			url: 'index.php?route=catalog/product.autocomplete&user_token={{ user_token }}&filter_name=' +  encodeURIComponent(request),
```

Comment:
@abdulwahidsharief , is there any method to form the url? this looks like hardcoded and any changes in product may break it.

Reply:
@ramth05 didn't find any method to get product catalog url

Reply:
changed to catalog_product_url